doc/releases/release-notes-2.1.rst

```diff
@@ -26,12 +26,21 @@ Architectures
 Boards & SoC Support
 ********************
 
-* TBD
+* CAN support:
```

Comment:
why is this directly under Boards & SoC Support?

Reply:
Because its CAN support for a board.
Tho other one is a shield.